add more font projects (#27)
Jolg42/awesome-typography: README.md

- [The OpenType Cookbook](http://opentypecookbook.com) - Introduction to OpenType features for type designers. [(Source)](https://github.com/typesupply/opentype-feature-intro).
- [A bit of font generation](https://pomax.github.io/CFF-glyphlet-fonts/) - Brief excursion into OpenType.

## Assembly
- [Fontraption](https://github.com/viler-int10h/Fontraption) - A tiny but powerful VGA text mode font editor for DOS
- [CGA-font-compare](https://github.com/viler-int10h/CGA-font-compare) - DOS tool to check if your CGA card uses the same ROM font as the original IBM CGA
- [oldschool-pc-fonts](https://int10h.org/oldschool-pc-fonts) - The world's biggest collection of classic text mode fonts, system fonts and BIOS fonts from DOS-era IBM PCs and compatibles - preserving raster typography from pre-GUI times.
- [x86-Assembly-Font-Renderer](https://github.com/mchlmmc/x86-Assembly-Font-Renderer) - Bootloader that draws characters from a 5x7px font.

> - [atari-font as assemlby source](https://forums.atariage.com/topic/267380-atari-font-as-assembly-source-file)

Reviewer: ```suggestion
- [atari-font as assembly source](https://forums.atariage.com/topic/267380-atari-font-as-assembly-source-file)
```

- [x86-Assembly-Fonts](https://github.com/ArTicZera/x86-Assembly-Fonts) - Use different bitmap fonts in your Real Mode OS using bitmaps and pixel routines.
- [xfnt](https://github.com/hexagonix/xfnt) - Fontes gráficas para Hexagon/Hexagon graphic fonts.
- [cracker](https://github.com/shred/cracker) - Text, graphics and font ripper for the ZX Spectrum.
- [persian-assembly-font](https://github.com/imnr/persian-assembly-font) -  Design and implementation of Persian alphabet letters in Assembly.
- [again](https://github.com/samoylenko/again) - My custom DirectX/Assembly/Raster-font PC Intro engine (2005).
- [amigafonts](https://github.com/rewtnull/amigafonts) - Faithfully remade multi platform Amiga fonts in Amiga aspect.
- [font-topaz-ng](https://codeberg.org/ideasman42/font-topaz-ng) - Amiga's system font, vectorized for modern systems.

## Shell
- [Crimson Pro](https://github.com/Fonthausen/CrimsonPro) - The Crimson Text typeface.
- [fontpreview-tui](https://codeberg.org/DCoderUltra/fontpreview-tui) - Minimal command line tool that allows to visualize fonts inside the terminal and copy the selected font to the clipboard.

> - [NerdFetch](https://codeberg.org/ThatOneCalculator/NerdFetch) - A POSIX *nix (Linux, macOS, Android, BSD, etc) fetch script using Nerdfonts (and others).

Reviewer: ```suggestion
- [NerdFetch](https://github.com/thatonecalculator/NerdFetch) - A POSIX *nix (Linux, macOS, Android, BSD, etc) fetch script using Nerdfonts (and others).
```
The GitHub repo seems better as it has a newer version compared to the repo on Codeberg.

- [TeensieFont](https://github.com/roundsToThree/TeensieFont) - A tiny font library intended for use on Arduino and other similar platforms.
- [craftmicro-fonts](https://github.com/craftmicro/craftmicro-fonts) - Pixel/bitmap font library for Craft Micro SDK. 
- [libint10h_fonts](https://github.com/cellularmitosis/libint10h_fonts) - The int10h.org bitmap font collection as a library of C structs.
- [ttf2mesh](https://github.com/fetisov/ttf2mesh) - Standalone library for TrueType font tessellation. Allows to load ttf-file and convert its glyphs to 2D or 3D mesh objects without rasterization.
- [otf2ttf](https://codeberg.org/zzkt/otf2ttf) - convert an OTF font to TTF.
- [psftools](https://codeberg.org/gnarz/psftools) - a simple textfile based psf font editor suite.
- [font-config-info](https://codeberg.org/derat/font-config-info) - Print Linux font configuration.

## C++
- [font to svg](https://github.com/donbright/font_to_svg) - Render characters from font files into an SVG path.
- [LCDF Typetools](https://github.com/kohler/lcdf-typetools) - Utilities for manipulating OpenType, PostScript Type 1, and Multiple Master fonts.

> - [HarfBuzz](https://github.com/googlei18n/harfbuzz) - HarfBuzz OpenType text shaping engine.
> - [FontView](https://github.com/googlei18n/fontview) - Demo app that displays fonts with a free/libre/open-source text rendering stack: FreeType, HarfBuzz and Raqm.

Reviewer: ```suggestion
- [HarfBuzz](https://github.com/harfbuzz/harfbuzz) - HarfBuzz OpenType text shaping engine.
- [FontView](https://github.com/googlefonts/fontview ) - Demo app that displays fonts with a free/libre/open-source text rendering stack: FreeType, HarfBuzz and Raqm.
```
Unrelated to the changes: fixing 2 links while reviewing :)

- [Text rendering tests](https://github.com/unicode-org/text-rendering-tests) - Test suite for text rendering.
- [FTGLES](https://github.com/cdave1/ftgles) - TrueType font rendering library for OpenGL ES on iOS devices (iPad and iPhone).
- [TTFPatch](https://github.com/rmuch/ttfpatch) - Mirror of TTFPATCH by Wolfram Esser, modified to support fonts conforming to newer OTF specifications, built for modern versions of Windows.
- [stb-truetype-opengl-examples](https://github.com/0xc0dec/demos) - Examples of TrueType font rendering in C++11 using stb_truetype library and OpenGL 3+.
- [WOFF2](https://github.com/google/woff2)
- [vectorfontstorm](https://github.com/VoxelStorm-Ltd/vectorfontstorm) - C++ vector font rendering library for OpenGL 3D text rendering, by VoxelStorm Resources

> - [Slug library](https://sluglibrary.com/) - Dynamic GPU Font Rendering and Advanced Text Layout

Reviewer: I think we should remove this, as there is no code there, it looks like a landing page to sell a product.
We don't have links like that at the moment.
```suggestion
```

Reviewer: Or we could move it to "Miscellaneous" and somehow mention that this is a paid product.

Author: There is a demo with shader source and thesis links

Author: https://sluglibrary.com/SlugDemo.zip

- [trex](https://github.com/KyrietS/trex) - Font rendering, atlas generation and text shaping library written in C++
- [ascii-art](https://github.com/codewithnick/ascii-art) - A C++ library to make everyday alphabets look much better on the terminal , this project uses OOPS concepts to make unique fonts and display letters on command line.
- [DigiFont](https://github.com/cbm80amiga/DigiFont) - Library for nice, scalable digit rendering using any Arduino GFX libraries.
- [artery-font-format](https://github.com/Chlumsky/artery-font-format) - Artery Atlas Font format library.
- [bmfont](https://sourceforge.net/projects/bmfont) - Bitmap font generator.
- [psfmaker](https://codeberg.org/CodePoet2005/psfmaker) - Tool for making linux console fonts. Output psf files and c header files.

> - [bdfreader](https://codeberg.org/heu/bdf-reader) - BDF font format reader.

Reviewer: I think that "bdfreader" should be moved to the Rust category

Author: Yes bdf-reader should be moved to the rust category
